keyboards/matrix/pjf/pjf.c

```diff
@@ -0,0 +1,22 @@
+/**
+ * pjf.c
+ */
+
+#include "pjf.h"
+
+__attribute__((weak))
+void matrix_init_user(void) {
```

Comment:
This should be the keyboard version.  And that way, it doesn't need to be weak
```suggestion
void matrix_init_kb(void) {
```